server/middleware/populateCurrentUser.ts

```diff
@@ -72,10 +72,11 @@ export default function populateCurrentUser(userService: UserService, licenceSer
             cvlUser.telephoneNumber = probationUser?.telephoneNumber
             cvlUser.probationAreaCode = probationUser?.probationArea?.code
             cvlUser.probationAreaDescription = probationUser?.probationArea?.description
-            cvlUser.probationPduCodes = probationUser?.teams?.map(team => team?.borough?.code)
-            cvlUser.probationLauCodes = probationUser?.teams?.map(team => team?.district?.code)
-            cvlUser.probationTeamCodes = probationUser?.teams?.map(team => team?.code)
-            cvlUser.probationTeams = probationUser?.teams
+            const teams = probationUser?.teams?.filter(t => !t.endDate || parseIsoDate(t.endDate) > new Date()) || []
```

Comment:
this will also "fix" the issue when someone has no probation teams - rather than receiving an unexpected error they will see no cases.. we might want to add some content here.